### PR TITLE
Removed Nuxt3 from intro

### PR DESCRIPTION
**Pull request by StirStudios**
Removed from the intro.

### CODE REVIEW

File: apps/docs/src/index.md [hunk 1]
```diff
@@ -38,15 +37,15 @@ Current Version
 With more than 35 components, several directives and composibles (and growing), BootstrapVueNext provides one of the most comprehensive implementations of the Bootstrap v5 component and grid system available for Vue.js v3 / Nuxt.js 3, complete with extensive and automated WAI-ARIA accessibility markup.
 
 <div class="d-flex gap-2 mt-4">
-  <b-button to="/docs" variant="primary">Get Started</b-button>
+  <b-button to="docs" variant="primary">Get Started</b-button>
   <b-button to="https://github.com/bootstrap-vue-next/bootstrap-vue-next" variant="outline-secondary">GitHub</b-button>
```
Comment (VividLemon) [inline apps/docs/src/index.md:41]: Utilize the inject value rather than having this link be hardcoded. An example exists in Layout.vue

**Review by VividLemon — CHANGES_REQUESTED** [review]

File: apps/docs/src/index.md (hunk 1 above)
Comment (StirStudios) [reply apps/docs/src/index.md:43]: Fixed both and we are rocking!

Comment (VividLemon) [reply apps/docs/src/index.md:43]: The first issue is not quite what I meant. The issue is you should use
```ts
const globalData = inject(appInfoKey, {
  discordUrl: '',
  githubUrl: '',
  opencollectiveUrl: '',
})
```
`:to="globalData.githubUrl"` instead

Comment (VividLemon) [reply apps/docs/src/index.md:43]: Ah, I see the issue. 

MD files will automatically compile markdown links: `An image` with the base, but will not automatically compile the html links. So in this context, withBase is the preferred fix

https://vitepress.dev/guide/asset-handling#base-url

Comment (StirStudios) [reply apps/docs/src/index.md]: Github link fixed now!

**Review by VividLemon — CHANGES_REQUESTED** [review]

File: apps/docs/src/index.md [hunk 2]
```diff
@@ -58,6 +59,13 @@ import {
...
   BRow,
 } from 'bootstrap-vue-next'
+import {inject} from 'vue'
+import {withBase} from 'vitepress'
 import BootstrapIcon from '~icons/simple-icons/bootstrap'
 import VueIcon from '~icons/simple-icons/vuedotjs'
+import {appInfoKey} from '../.vitepress/theme/keys'
+
+const globalData = inject(appInfoKey, {
+  githubUrl: '',
```
Comment (VividLemon) [inline apps/docs/src/index.md:69]:
```suggestion
  discordUrl: '',
  githubUrl: '',
  opencollectiveUrl: '',
```

File: apps/docs/src/index.md [hunk 3]
```diff
@@ -30,23 +29,25 @@ Integrate with Nuxt.js v3
...
 
 ## Overview
 
 With more than 35 components, several directives and composibles (and growing), BootstrapVueNext provides one of the most comprehensive implementations of the Bootstrap v5 component and grid system available for Vue.js v3 / Nuxt.js 3, complete with extensive and automated WAI-ARIA accessibility markup.
 
 <div class="d-flex gap-2 mt-4">
-  <b-button to="/docs" variant="primary">Get Started</b-button>
-  <b-button to="https://github.com/bootstrap-vue-next/bootstrap-vue-next" variant="outline-secondary">GitHub</b-button>
+  <b-button :to="withBase('/docs')" variant="primary">Get Started</b-button>
+  <b-button :to="globalData.githubUrl" variant="outline-secondary">GitHub</b-button>
```
[the remark this answers is not in the document]
Comment (StirStudios) [reply apps/docs/src/index.md]: Agreed, as I go through will updated as needed. I have been spoiled with nuxt js taking care of that!

**Pull request merged**